src/psyclone/tests/dynamo0p3_ref_elem_test.py

```diff
@@ -98,7 +107,7 @@ def test_mdata_invalid_property():
     with pytest.raises(ParseError) as err:
         DynKernMetadata(ast, name=name)
     assert ("property: 'not_a_property'. Supported values are: "
-            "['normals_to_horizontal_faces'," in str(err.value))
+            "['NORMALS_TO_HORIZONTAL_FACES'," in str(err.value))
```

Comment:
This can be lower case - at least ref_elem metadata are lower case in kernels. I guess here it is upper case just to check the parse error?

Reply:
Yep.